src/ggshield-webview/gitguardian-webview-view.ts

```diff
@@ -146,13 +142,59 @@ export class GitGuardianWebviewProvider implements vscode.WebviewViewProvider {
         </body>
         </html>
     `;
+    } else {
+      computedHtml = `
+        <!DOCTYPE html>
+        <html lang="en">
+        <head>
+          <meta charset="UTF-8">
+          <meta name="viewport" content="width=device-width, initial-scale=1.0">
+          <link href="${styleUri}" rel="stylesheet">
+          <title>GitGuardian - Authentication Error</title>
+        </head>
+        <body>
+          <div class="anonymous">
+            <h1>❌ Authentication failed.</strong></h1>
```

Comment:
I think we should add a link to the [configuration documentation](https://docs.gitguardian.com/ggshield-docs/configuration) here (also we might want to rework that doc but that's for an other day )